config/environments/production.rb

```diff
@@ -37,14 +37,14 @@
   # yet still be able to expire them through the digest params.
   config.assets.digest = true
 
-  config.paperclip_defaults = {
-      :storage => :s3,
-      :s3_credentials => {
-          :bucket => ENV['S3_BUCKET_NAME'],
-          :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-          :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
-      }
-  }
+config.paperclip_defaults = {
+     :storage => :s3,
+     :s3_credentials => {
+         :bucket => ENV['AWS_BUCKET'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.